src/test/resources/org/opencds/cqf/individual_tooling/cql_generation/drool/visitor/Abdominal_Cramps_314c4.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file (and others) are still under the "individual_tooling" package. Needs to move to "tooling"